test/Chirp.Infrastructure.Tests/RepositoryTests/AuthorRepositoryTest.cs

```diff
@@ -0,0 +1,261 @@
+
+        AuthorDTO theAuthor = null;
+        for (int i = 0; i < 34; i++)
+        {
+
+            if (i == 5)
+            {
+                theAuthor = new AuthorDTO
+                {
+                    AuthorId = Guid.NewGuid(),
+                    Name = "TestAuthor" + i,
+                    Email = "mock" + i + "@email.com"
+                };
+                context.Authors.Add(theAuthor);
+                continue;
+            }
+
+            AuthorDTO authorDto = new AuthorDTO
+            {
+                AuthorId = Guid.NewGuid(),
+                Name = "TestAuthor" + i,
+                Email = "mock" + i + "@email.com"
+            };
+
+            CheepDTO cheepDto = new CheepDTO
+            {
+                CheepId = Guid.NewGuid(),
+                AuthorId = authorDto.AuthorId,
+                Text = "TestCheep" + i,
+                AuthorDto = authorDto
+            };
+
+            context.Authors.Add(authorDto);
+            context.Cheeps.Add(cheepDto);
+        }
+
+        context.SaveChanges();
+
+        //Act
+        AuthorDTO expectedAuthor = theAuthor;
+        AuthorDTO returnedAuthor = authorRepository.GetAuthorByName("TestAuthor5");
+
+        //Assert
+        Assert.Equal(expectedAuthor, returnedAuthor);
+
+    }
+
+    [Fact]
+    public void GetCheepsByAuthor_ShouldReturnCorrectCheeps()
+    {
+
+        //Arrange
+        var authorRepository = new AuthorRepository(context);
+
+        AuthorDTO authorDto1 = new AuthorDTO
+        {
+            AuthorId = Guid.NewGuid(),
+            Name = "TestAuthor",
+            Email = "mock@email.com"
+        };
+        CheepDTO cheepDto1 = new CheepDTO
+        {
+            CheepId = Guid.NewGuid(),
+            AuthorId = authorDto1.AuthorId,
+            Text = "TestCheep",
+            AuthorDto = authorDto1
+        };
+
+        context.Authors.Add(authorDto1);
+        context.Cheeps.Add(cheepDto1);
+
+        context.SaveChanges();
+
+        //Act
+        ICollection<CheepDTO> expectedCheep = new List<CheepDTO>();
+        expectedCheep.Add(cheepDto1);
+
+        ICollection<CheepDTO> returnedCheep = authorRepository.GetCheepsByAuthor(authorDto1.Name, 0);
+
+        //Assert
+        Assert.Equal(expectedCheep, returnedCheep);
+    }
+
+    [Fact]
+    public void addAuthor_ShouldAddAuthorToDatabase()
+    {
+        //Arrange
+        var authorRepository = new AuthorRepository(context);
+
+        AuthorDTO authorDto1 = new AuthorDTO
+        {
+            AuthorId = Guid.NewGuid(),
+            Name = "TestAuthor",
+            Email = "mock@email.com"
+        };
+        CheepDTO cheepDto1 = new CheepDTO
+        {
+            CheepId = Guid.NewGuid(),
+            AuthorId = authorDto1.AuthorId,
+            Text = "TestCheep",
+            AuthorDto = authorDto1
+        };
+
+        context.Authors.Add(authorDto1);
+        context.Cheeps.Add(cheepDto1);
+
+        context.SaveChanges();
+
+        // Act
```

Comment:
Why is the 'Act' part of this the same as the 'Arrange part'? Shouldn't authorDto1 and authorDto2 both be in 'Arrange'?

Reply:
Good point!
This is now fixed.